src/openms/source/CHEMISTRY/RibonucleotideDB.cpp

```diff
@@ -131,44 +165,53 @@ namespace OpenMS
       OPENMS_LOG_WARN << "Average mass of " << code << " differs substantially from its formula mass.\n";
     }
 
-    if (!(entry.at("mass_monoiso").is_null()))
+    if (!(entry.find("mass_monoiso") == entry.cend()) && !(entry.at("mass_monoiso").is_null()))
```

Comment:
borderline complexity of expression :)
the newer syntax for if expressions with initalization statement would make it easier/faster to parse for humans:
```
if (auto e = entry.find("mass_monoiso"); e != entry.cend() && !e->is_null)
{
}
```